code/modules/vehicles/cars/clowncar.dm

```diff
@@ -110,10 +110,32 @@
 
 /obj/vehicle/sealed/car/clowncar/Bump(atom/bumped)
 	. = ..()
-	if(isliving(bumped) && !istype(bumped, /mob/living/basic/deer))
+	if(isliving(bumped))
 		if(ismegafauna(bumped))
 			return
 		var/mob/living/hittarget_living = bumped
+
+		if(istype(hittarget_living, /mob/living/basic/deer))
+			visible_message(span_warning("[src] careens into [hittarget_living]! Oh the humanity!"))
+			for(var/mob/living/carbon/carbon_occupant in occupants)
+				if(prob(65)) //Note: The randomstep on dump_mobs throws occupants into each other and often causes wounds regardless.
```

Comment:
make this a `prob(35)` and early continue instead this is some wacky indentation we don't need